shared/modules/logger_module.go

```diff
@@ -0,0 +1,20 @@
+package modules
+
+//go:generate mockgen -source=$GOFILE -destination=./mocks/logger_module_mock.go -aux_files=github.com/pokt-network/pocket/shared/modules=module.go
+import (
+	"github.com/rs/zerolog"
+)
+
+type Logger = zerolog.Logger
+
+type LoggerModule interface {
+	Module
```

Comment:
I guess `LoggerModule` doesn't have to embed all the interfaces from `Module` since most of them are unused/no-ops.

We can always revisit it once we merge in the config/genesis PR, as we speak, this is how `module.go` looks like https://github.com/pokt-network/pocket/pull/235/files#diff-ae3949fd3b550b6789e9ec9f0557c73dc0b3eb99b91048bf8ad4406a0b77f1abR9

Reply:
If we do that, then this signature is going to be violated: `	Create(runtime RuntimeMgr) (Module, error)`

Reply:
We discussed this offline. Create() can be changed to return `InitializableModule` instead of `Module`. Ticket for this change: https://github.com/pokt-network/pocket/issues/333